packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -179,6 +179,27 @@ const Conversation = WebexPlugin.extend({
       });
   },
 
+  /**
+   * react to an activity
+   * @param {Object} conversation
+   * @param {Object} object
```

Comment:
What is the expected shape of the object? Could we rename `object` to something a bit more descriptive?

Reply:
ill update the comment and rename!